app/models/package_manager/base.rb

```diff
@@ -92,25 +92,43 @@ def self.map_project(project)
         &.delete_if { |_key, value| value.blank? }
     end
 
-    def self.update_version(name, version)
-      unless self::SUPPORTS_SINGLE_VERSION_UPDATE
-        logger.warn("#{db_platform}.update_version(#{name}, #{version}) called but not supported on platform")
+    def self.update(name, sync_version: :all)
+      if sync_version != :all && !self::SUPPORTS_SINGLE_VERSION_UPDATE
+        logger.warn("#{db_platform}.update(#{name}, sync_version: #{sync_version}) called but not supported on platform")
```

Comment:
Should we raise here instead?

Reply:
Warn seems fine, it's not really an error I think. It should honestly just silently *not* do a version update, then we don't have to inspect that logic anywhere but here.